src/JunctionTreeTypes.jl

```diff
@@ -14,14 +14,17 @@ mutable struct BayesTree
   btid::Int
   cliques::Dict{Int,Graphs.ExVertex}
   frontals::Dict{Symbol,Int}
+  #TEMP JT for evaluation, to store message channels in tree, would be better stored on edges
```

Comment:
personally im not there yet for messages on edges, but havent worked with it enough to know how existing CSM workflows will have to change to accommodate both cliques and edges.  Previous workflow was to only use cliques and that became hairy enough.  The suggestion of "hiding" where the messages are stored through accessor API functions might work, and will have to watch out for debug and step processes in CSM dev where cliques are being cloned (but edges are currently not being cloned).

Reply:
Note, it is just the message channel associated with the edge (there is no more information added, the relationship already exists). The message data is stored in CSM.
So there is no need to clone edges.

Reply:
refer to #459 for further discussion.